docs/website/versioned_docs/version-3.0.0/command-reference/dev.md

```diff
@@ -30,7 +30,7 @@ $ odo dev
  ✓  Executing the application [1s]
 
 Your application is now running on the cluster
- - Forwarding from 127.0.0.1:40001 -> 3000
+ - Forwarding from 127.0.0.1:40001 -3000
```

Comment:
Why? This has not been removed from the code, why are you removing this from the docs?